ourpackage/piadozo.py

```diff
@@ -0,0 +1,99 @@
+import os
+import random
+
+def edriane():
+    while True:
+        os.system('cls')
+        print ("Your Teammate :)")
+        print ("1. Who am I ?")
+        print ("2. What's your thought to me as a teammate :>")
+        print ("3. Rate my function")
+        print ("0. Exit")
```

Comment:
I will add my comment/changes. - Barcelos

Reply:
You have done it :)